wurst/file/legacy/LocalFiles.wurst

```diff
@@ -16,6 +16,10 @@ interface LocalFilesCallback
 	In other words, this checks if the user can reliably write/read files.
 	This also includes a fallback mechanism for the 1.26 patch and before,
 	which uses the Logs\ folder to write files in.
+
+	NOTE: Since patch 1.30, the requirement for the Allow Local Files registry
+	setting has been lifted. This class will always succeed for future versions.
```

Comment:
not always probably.

Reply:
@Frotty What makes you assume this wouldn't always be the case?

Reply:
because the localfiles requirement has been lifted with 1.30
It will be deprecated and removed eventually.